sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSessionManager.java

```diff
@@ -298,13 +298,17 @@ private AmqpErrorContext getErrorContext() {
                 if (existing != null) {
                     return existing;
                 }
+
                 return new ServiceBusSessionReceiver(link, messageSerializer, connectionProcessor.getRetryOptions(),
                     receiverOptions.getPrefetchCount(), disposeOnIdle, scheduler, this::renewSessionLock,
                     maxSessionLockRenewDuration);
             })))
             .flatMapMany(sessionReceiver -> sessionReceiver.receive().doFinally(signalType -> {
-                logger.verbose("Adding scheduler back to pool.");
+                logger.verbose("Adding scheduler back to pool for session-id [{}].", sessionReceiver.getSessionId());
```

Comment:
This log message should instead be `"Closing session receiver for session id {}"`. "Adding scheduler back to pool" is not a very useful debug message for the customer or for us to troubleshoot.